website/.env.dist

```diff
@@ -1,17 +1,17 @@
-SESSION_SECRET=your_secret_key
+
 POCKETBASE_ORIGIN=http://localhost:3000
-PUBLIC_POCKETBASE_ORIGIN= http://localhost:3000
+PUBLIC_POCKETBASE_ORIGIN=http://localhost:3000
 POCKETBASE_EMAIL=your_email
 POCKETBASE_PASSWORD=your_password
-RESEND_API_KEY=your_api_key
 ORIGIN=http://localhost:3000
-LUMA_API_KEY=your_api_key
-ZAPIER_WEBHOOK_SECRET=your_webhook_secret
-POSTHOG_PUBLIC_API_KEY=your_api_key
-SENTRY_DSN=your_dsn
-SENTRY_AUTH_TOKEN=your_auth_token
-SENTRY_ORG=your_org
-SENTRY_PROJECT=your_project
-INNGEST_ENABLED=true
-INNGEST_SIGNING_KEY=your_signing_key
-INNGEST_EVENT_KEY=your_event_key
+
+# Optionals 
+# SESSION_SECRET=
```

Comment:
I think it's required: `sessionSecret: z.string().min(1),`

Reply:
it is but it is set automatically if not provided.

`sessionSecret: process.env.SESSION_SECRET || 'local-session-secret-' + crypto.randomUUID(),`